tests/static/intentManager/printing-intent.xml

```diff
@@ -24,7 +24,7 @@
 		</model>
 		<odkx:intent xmlns:odkx="http://opendatakit.org/xforms" id="print_data" class="org.commcare.dalvik.action.PRINT">
 			<extra key="text" ref="/data/text" />
-			<extra key="cc:print_template_reference" ref="jr://file/commcare/doc/data/print_data.doc" />
+			<extra key="cc:print_template_reference" ref=".">jr://file/commcare/doc/data/print_data.doc</extra>
```

Comment:
This won't work, we don't process this as an XPath instance, we just parse it out when we parse the form.